services/auth/office/helpers/getValidSigningKey.js

```diff
@@ -0,0 +1,15 @@
+import certToPEM from './certToPEM';
+
+import { JWK } from '../constants';
+
+export default function getValidSigningKey({ use, kty, kid, x5c = [], n, e }) {
```

Comment:
These abbreviations (kty, kid, x5c) might be hard to understand if you have no or little auth experience. Could we add a comment that points to some form of documentation to ease the understanding? :)